src/DotNetWorker.Core/Context/Features/DefaultFunctionInputBindingFeature.cs

```diff
@@ -72,25 +72,12 @@ public async ValueTask<FunctionInputBindingResult> BindFunctionInputAsync(Functi
                     {
                         var properties = new Dictionary<string, object>();
 
-                        // Pass info about specific input converter type defined for this parameter, if present.
-                        if (param.Properties.TryGetValue(PropertyBagKeys.ConverterType, out var converterTypeAssemblyFullName))
-                        {
-                            properties.Add(PropertyBagKeys.ConverterType, converterTypeAssemblyFullName);
-                        }
-
-                        // Pass info about the flag to allow fallback to default converters defined for this parameter, if present.
-                        if (param.Properties.TryGetValue(PropertyBagKeys.AllowConverterFallback, out var flag))
-                        {
-                            properties.Add(PropertyBagKeys.AllowConverterFallback, flag);
-                        }
-
-                        // Pass info about input converter types defined for this parameter, if present.
-                        if (param.Properties.TryGetValue(PropertyBagKeys.BindingAttributeSupportedConverters, out var converters))
-                        {
-                            properties.Add(PropertyBagKeys.BindingAttributeSupportedConverters, converters);
-                        }
-
-                        var converterContext = _converterContextFactory.Create(param.Type, source, context, properties.Count() != 0 
+                        AddFunctionParameterPropertyIfPresent(properties, param, PropertyBagKeys.ConverterType);
+                        AddFunctionParameterPropertyIfPresent(properties, param, PropertyBagKeys.AllowConverterFallback);
+                        AddFunctionParameterPropertyIfPresent(properties, param, PropertyBagKeys.BindingAttributeSupportedConverters);
+                        AddFunctionParameterPropertyIfPresent(properties, param, PropertyBagKeys.BindingAttribute);
```

Comment:
The existing properties are primarily there to support infrastructure, but since this one will be exposed to converters, we should create an extension method for `ConverterContext` so that is what is ultimately used by the converters, abstracting away the details of how this is added to the context and improving discoverability.

Reply:
Just to clarify, are you suggesting a ConverterContext extension method that is used directly by the converter to grab the BindingAttribute i.e. `context.GetBindingAttribute()`. But no change to this file as we still need to pass that information down to the ConverterContext in the first place, or are you also suggesting that this extension method does what AddFunctionParameterPropertyIfPresent is doing here too?

Reply:
No change to this file. Just the public API that would be consumed by the converters.

Reply:
@fabiocav I think this is addressed, mind reviewing again? Thanks!